android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactTVExoplayerViewManager.java

```diff
@@ -269,6 +270,11 @@ public void setStats(final ReactTVExoplayerView videoView, final boolean hasStat
         videoView.setStats(hasStats);
     }
 
+    @ReactProp(name = PROP_CONTROLS)
+    public void setControls(final ReactTVExoplayerView videoView, final boolean visible) {
+        videoView.setControls(visible);
```

Comment:
should this be called showControls ?

Reply:
Nope. This way is consistent with how the current API is defined with `set`/`get` as prefixes.